dev-libs/OpenNI2/OpenNI2-2.2_beta2-r1.ebuild

```diff
@@ -9,7 +9,7 @@ if [ "${PV#9999}" != "${PV}" ] ; then
 	EGIT_REPO_URI="https://github.com/occipital/openni2"
 fi
 
-inherit ${SCM} toolchain-funcs multilib java-pkg-opt-2 flag-o-matic
```

Comment:
OpenNI2 at least has a bug open for this, so we should tag it: https://bugs.gentoo.org/694070